lib/pwa.js

```diff
@@ -198,8 +198,12 @@ exports.createOrUpdatePwa = function(pwa) {
       }
       // Delete modified PWA from cache
       const url = '/pwas/' + pwa.id;
-      cache.del(url);
-      cache.del(url + '?contentOnly=true');
+      cache.del(url).catch(_ => {
+        // URL not in cache.
```

Comment:
Should the catch handler do something, such as logging? Otherwise, we can just remove the catch block.